src/components/popDiv/popDiv.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState, useEffect, useRef } from "react";
+import "../../data/styles/common.scss";
+import "./index.scss";
+import UserMsg from "../userMsg/userMsg";
+// import BotHeader from "../botHeader/botHeader";
+import HighlightOffIcon from "@material-ui/icons/HighlightOff";
+import FiberManualRecordIcon from '@material-ui/icons/FiberManualRecord';
+
+const botMessage = [
+  { id: 1, text: "Hi, I am your virtual help bot." },
+  { id: 2, text: "How can I help you today?" }
+];
+
+function PopDiv() {
+  const [close, closeBot] = useState(false);
+
+  //saving the text of input area
+  const [inputText, setInputText] = useState("");
+
+  //for messages the user sends
+  const [msgs, setMsgs] = useState([]);
+
+  const chatbodyRef = useRef();
+  //reference created
+
+  const scrollToBottom = () => {
+    chatbodyRef.current.scrollIntoView({ behavior: "smooth" });
+  };
+  //scrollToBottom function
+
+  useEffect(scrollToBottom, [msgs]);
```

Comment:
useEffect(()=>{
chatbodyRef.current.scrollIntoView({ behavior: "smooth" });
},[msgs]

do not assign fund name like scrollToBottom